src/uiveri5.js

```diff
@@ -318,6 +318,25 @@ function run(config) {
         });
       };
 
+      browser.get = function (sUrl, iTimeout) {
```

Comment:
what about the authentication ?

Reply:
it's not always needed to authenticate; we could have another method for programmatic authentication

Reply:
1. no need for an explicit iTimeout setting, the global settings will apply.
2. Lets have a single .get method with optional auth param that defaults to plain. It should call navigate.to as we should keep it for backward compatibility

Reply:
the auth param is supposed to contain the credentials. if a default strategy is used, or even if auth is just "enabled", credentials could be read from the config file but that would be very confusing to the user imo
navigate.to has some logging specific to the first page load. do we need browser.refresh/sleep  and version stamps in browser.get as well?

Reply:
the auth param should contain the auth configuration that may include the credentials. Exactly the same as navigate.to with the difference that empty config = plain

Reply:
I suggest that we directly call navigate.to with a default value if auth is missing